beeline/__init__.py

```diff
@@ -128,7 +128,22 @@ def add(data):
 
 
 def tracer(name):
-    return g_tracer(name)
+    '''
+    When used in a context manager, creates a trace event for the contained
+    code. If existing trace context data is available, will mark the event as a
+    child of the most recent trace span.
+
+    Example use:
+
+    ```
+    with tracer(name="call to db"):
```

Comment:
nit: to match the example in readme, this could could be `with tracer("call to db"):`
https://github.com/honeycombio/beeline-python/blame/master/README.md#L211